### PR TITLE
[4.0] RTL: Correcting Header More Menu and Toggle Burger

### PR DESCRIPTION
**Pull request by infograf768**
### Summary of Changes 

Adding specific RTL css to Atum

### Testing Instructions
Install a new site and add Persian language (fa-IR).
Load `administrator/index.php?option=com_languages&view=installed&client=1` (admin languages)
Switch to Persian.

Reduce browser window width as shown below, click on the Header More Icon (Ellipsis icon) to display the various modules in the Status position as they are hidden when the window is reduced.

**To patch, one has to use npm** !!!

### Before patch
badheader

### After patch

goodheader

### Note
There are many other aspects of RTL to be taken care of when reducing the window width.

### CODE REVIEW

File: administrator/templates/atum/scss/blocks/_header.scss [hunk 1]
```diff
@@ -243,10 +249,20 @@
...
+
+      [dir=rtl] & {
+        left: 0;
+        right: auto;
+      }
     }
     &.active .header-more-menu {
       left: auto;
+
+     [dir=rtl] & {
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_header.scss]:
```suggestion
      [dir=rtl] & {
```

Comment (infograf768) [reply administrator/templates/atum/scss/blocks/_header.scss]: done.

**Comment by vaibhavsTekdi** [comment]
I have tested this item :red_circle: unsuccessfully on 9d10b058cd723fafa79b464e7bce2d11232ad34f<br /><br />Tested this with the patch but it's not working as per expectations. Placement is working but the toggle is not working for multilingual site.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26613">issues.joomla.org/tracker/joomla-cms/26613</a>.</sub>

**Comment by pallaviu** [comment]
I have tested this item :red_circle: unsuccessfully on 9d10b058cd723fafa79b464e7bce2d11232ad34f<br /><br />Menu placement in multilingual is correct but toggle is not working as expected.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26613">issues.joomla.org/tracker/joomla-cms/26613</a>.</sub>

**Comment by infograf768** [comment]
This patch needs using npm to create the correct css.

**Comment by infograf768** [comment]
@pallaviu @vaibhavsTekdi 

Are you testing with patchtester or from a github clone and composer/npm ?

You need the later to test.

**Comment by sanderpotjer** [comment]
I have tested this item :white_check_mark: successfully on 44809c0921f12f91da48475c762d561ebe35750c<br /><br />After npm the mentioned RTL issues are fixed.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26613">issues.joomla.org/tracker/joomla-cms/26613</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**